graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/journal/RawMessage.java

```diff
@@ -166,7 +167,7 @@ public int getVersion() {
     }
 
     public DateTime getTimestamp() {
-        return new DateTime(msgBuilder.getTimestamp()); // TODO PERFORMANCE object creation
+        return new DateTime(msgBuilder.getTimestamp(), DateTimeZone.UTC); // TODO PERFORMANCE object creation
```

Comment:
As far as I can see this is fine. This method is only used in `RawMessage#toString()` and in some codecs to retrieve the message timestamp. Since we pass around a complete `DateTime` object (which explicitly contains a timestamp), this won't change anything (famous last words).